[AN] 내부공유 CICD 빌드 실패하는 문제 해결

### 버그 설명

```
Run if [ -n "" ]; then
  if [ -n "" ]; then
    VERSION_NAME=""
  else
    VERSION_NAME=$(./gradlew -q printVersionName)
  fi
  echo "VERSION_NAME=$***VERSION_NAME***" >> $GITHUB_ENV
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail ***0***
  env:
    GOOGLE_SERVICES_JSON: ***
    LOCAL_PROPERTIES_CONTENTS: ***
    JAVA_HOME: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/21.0.8-9/x64
    JAVA_HOME_21_X64: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/21.0.8-9/x64
    VERSION_CODE: 125082253

FAILURE: Build failed with an exception.

* Where:
Build file '/home/runner/work/2025-bottari/2025-bottari/android/Bottari/data/build.gradle.kts' line: 19

* What went wrong:
RELEASE_BASE_URL is missing in local.properties

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org./

BUILD FAILED in 20s
```

### 재현 방법

an/develop에 push

### 환경 정보

_No response_

### 체크리스트

- [x] 최신 버전에서도 발생하는 문제인지 확인했습니다.
- [x] 중복된 이슈가 없는지 확인했습니다.

--- a/.github/workflows/android-develop-cI-cd.yaml
+++ b/.github/workflows/android-develop-cI-cd.yaml
@@ -58,9 +58,11 @@ jobs:
 
       - name: Get versionName
         run: |
-          if [ -n "${{ github.event.inputs.versionName }}" ]; then
+          if [ "${{ github.event.inputs.versionName != '' }}" = "true" ]; then
+            echo "✅ versionName provided: '${{ github.event.inputs.versionName }}'"
             VERSION_NAME="${{ github.event.inputs.versionName }}"
           else
+            echo "ℹ️ versionName not provided, using Gradle output..."
             VERSION_NAME=$(./gradlew -q printVersionName)
           fi
           echo "VERSION_NAME=${VERSION_NAME}" >> $GITHUB_ENV

